docs/core/deploying/index.md

```diff
@@ -32,23 +33,28 @@ Executables aren't cross-platform, they're specific to an operating system and C
 
 The following commands produce an executable:
 
-| Type                                                                                     | Command |
-| ---------------------------------------------------------------------------------------- | ------- |
-| [framework-dependent executable](#publish-framework-dependent) for the current platform. |  [`dotnet publish`](../tools/dotnet-publish.md) |
-| [framework-dependent executable](#publish-framework-dependent) for a specific platform.  |  [`dotnet publish -r <RID>`](../tools/dotnet-publish.md) |
-| [self-contained executable](#publish-self-contained).                                    |  [`dotnet publish -r <RID> --self-contained`](../tools/dotnet-publish.md) |
+| Type | Command |
+|--|--|
+| [framework-dependent executable](#publish-framework-dependent) for the current platform. | [`dotnet publish`](../tools/dotnet-publish.md) |
+| [framework-dependent executable](#publish-framework-dependent) for a specific platform. | [`dotnet publish -r <RID>`](../tools/dotnet-publish.md) |
+| [self-contained executable](#publish-self-contained). | [`dotnet publish -r <RID> --self-contained`](../tools/dotnet-publish.md) |
 
 ## Produce a cross-platform binary
 
 Cross-platform binaries are created when you publish your app as [framework-dependent](#publish-framework-dependent), in the form of a *dll* file. The *dll* file is named after your project. For example, if you have an app named **word_reader**, a file named *word_reader.dll* is created. Apps published in this way are run with the `dotnet <filename.dll>` command and can be run on any platform.
 
 Cross-platform binaries can be run on any operating system as long as the targeted .NET runtime is already installed. If the targeted .NET runtime isn't installed, the app may run using a newer runtime if the app is configured to roll-forward. For more information, see [framework-dependent apps roll forward](../versions/selection.md#framework-dependent-apps-roll-forward).
 
+If you're running the app in a container, you can choose to run the app directly with OS as the app host or with the `dotnet` as the host. There should be no app behavior difference when launching from via app host versus the `dotnet` host for ordinary server apps. Launching via an app host gives you better integration with the underlying OS. For example:
```

Comment:
This choice is not specific to containers. "If you're running the app in a container," is superfluous and confusing..